packages/graph-explorer/src/connector/gremlin/mappers/mapApiEdge.ts

```diff
@@ -5,15 +5,23 @@ import parseEdgePropertiesValues from "./parseEdgePropertiesValues";
 import { extractRawId } from "./extractRawId";
 
 const mapApiEdge = (apiEdge: GEdge): Edge => {
+  // TODO: Is this problematic for edges that naturally have no properties? Is
+  // there another way to tell if the Edge actually has zero properties?
```

Comment:
I'm leaving this TODO in here. The branch with the actual query editor logic deals more with the `isFragment` logic, so I'll resolve it there.